packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "netapp_snapshot_policies_create": "Créer une nouvelle Snapshot policy",
+  "netapp_snapshot_policies_create_title": "Créer une nouvelle Snapshot policy",
```

Comment:
any reason why you changed the key? now you need to submit for translation again.

Reply:
The same key was present in the other page. So that was causing the issue. So renamed one of the repetitive key to solve the issue.